std/datetime/stopwatch.d

```diff
@@ -349,6 +349,13 @@ private:
     long _ticksElapsed;    // Total time that the StopWatch ran before it was stopped last.
 }
 
+private void doNotOptimizeAway(T)(auto ref T t) @trusted @nogc nothrow
+{
+    import core.thread : getpid;
+    import core.stdc.stdio;
+    if(getpid() == 1)
```

Comment:
@klickverbot Yah, but the stack is a free-for-all address space and I wonder whether the compiler acknowledges that. Meaning, do compilers assume a global may point to stack-allocated data (from a previous function) even though the current stack data does not escape?